tests/WebAppCallsMicrosoftGraph/Startup.cs

```diff
@@ -30,7 +30,8 @@ public void ConfigureServices(IServiceCollection services)
             services.AddAuthentication(OpenIdConnectDefaults.AuthenticationScheme)
                     .AddMicrosoftIdentityWebApp(Configuration.GetSection("AzureAd"))
                         .EnableTokenAcquisitionToCallDownstreamApi()
-                           .AddInMemoryTokenCaches();  // Add a delegate overload. Should return the parent builder
+                           .AddMicrosoftGraphServiceClient(Configuration.GetSection("GraphBeta"))
```

Comment:
nit: question for asp .net maybe...`.AddMicrosoftGraph`? or leave as is...i'm not sure. `...ServiceClient` is more precise